### ISSUE TITLE
CLI UI needs aesthetic improvements for users with light colored terminal backgrounds  

### ISSUE THREAD

**Issue by eliheuer** [googlefonts/fontbakery#2031]
## Observed behaviour

When using the CLI with a light colored background the information presented is hard to read and looks broken. See screenshot below:

![light-bg](https://user-images.githubusercontent.com/5162664/44668615-e8074780-a9eb-11e8-8b42-232762e342ba.png)

## Expected behaviour

Many(most?) type designers use MacOS which defaults to a white terminal background. I have observed that many type designers also don't spend most of their time using the terminal and never adjust the colors. Therefore, I propose that the graphic elements of the fontbakery CLI should be adjusted so they look aesthetically pleasing, or at least considered, to people using terminals with a light background.

I use a custom dark terminal theme for MacOS, and I would also like to see the forced dark background removed from things like the cupcake artwork. See screenshot below:

![cupcake-issue](https://user-images.githubusercontent.com/5162664/44667297-69f57180-a9e8-11e8-9285-460cb16a1401.png)

**Comment by eliheuer** [comment]
I might try to make a pull request to fix some of these issues at some point before ATypI. I'm looking at this project as an example of how to do this with python3: https://github.com/sdushantha/neofetch

Maybe defaulting to a set of colors that look nice on white and then having a config file for regular terminal users where then can switch a `dark_mode` flag to true would be the best solution?

**Comment by davelab6** [comment]
ANSI has default background color as a mode, f'sure

**Comment by felipesanches** [comment]
Yeah, this issue was evident among users at the ATypI 2018 Font Bakery workshop...

**Comment by kontur** [comment]
<img width="320" alt="Screenshot 2020-02-11 at 16 33 59" src="https://user-images.githubusercontent.com/1893577/74245852-57f6b400-4cec-11ea-8dac-90b103cb1698.png">

In iTerm (Build 3.3.9beta1) with fontbakery at 3caba0e47c6a9440c16fe911db74f1b2bcf393e9 the above is the output on a _dark_ terminal now.

**Comment by felipesanches** [comment]
you can use `--dark-theme`

**Comment by kontur** [comment]
Ah, okay, skimmed over the above too quick

> Maybe defaulting to a set of colors that look nice on white and then having a config file for regular terminal users to switch a dark_mode flag to true would be the best solution?

I can certainly live with it, but in this instance fontbakery is called as part of a build script, so I wouldn't overwrite it in there. Could be something for the fontbakery user settings in discussed in #2734

**Comment by m4rc1e** [comment]
@felipesanches it would be great if we could drop the need for `--dark-theme` this year. If pytest can do it so can we.
